tests/pom.xml

```diff
@@ -337,6 +339,7 @@
                 <!-- Change "container/folder" to contain the testing container  -->
                 <!-- Keep the structure in form container/folder -->
                 <containerKey>swift://container/folder</containerKey>
+                <alluxio.underfs.listing.length>50</alluxio.underfs.listing.length>
```

Comment:
Would it be possible to set this config value in the test instead of the profile?